src/workerd/io/actor-cache-test.c++

```diff
@@ -1427,70 +1413,122 @@ KJ_TEST("ActorCache read retry on flush containing only puts") {
   KJ_ASSERT(KJ_ASSERT_NONNULL(promise.wait(ws)) == "123");
 }
 
-KJ_TEST("ActorCache read hard fail") {
+// KJ_TEST("ActorCache read hard fail") {
```

Comment:
Since we've moved reads to flush along with writes, if the read fails, it looks like we're actually [breaking the output gate](https://github.com/cloudflare/workerd/pull/1814/files#diff-52b552cc9d6477516b49e970aa23de236c7a1f7327b09c243c336445cbb2e18eR2751-R2752) :grimacing:.

Reply:
😬